user/services/register_service.py

```
@@ -44,5 +45,5 @@ def send_verification_email(user: User):
             recipient_email=user.email,
         )
         email.send()
-    except Exception as e:
-        print(f"Failed to send verification email to {user.email}. Error: {e}")
+    except Exception:
+        raise EmailCanNotBeSentException()
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)